components/centraldashboard/public/components/dashboard-view.pug

```diff
@@ -8,7 +8,27 @@
                         .header [[item.text]]
                         aside(secondary) [[item.desc]]
                     paper-ripple
+template(is='dom-if', if='[[platformDetails.resourceChartsLink]]')
+    .column
```

Comment:
I don't think the column should be optional, only perhaps the chart cards within. What do you think @prodonjs @ajayalfred?

Reply:
My thought here was that if there was no metrics provider defined for the platform, the third column would become to the second column. It does create some unused space, but unless we were to abandon the grid layout in favor of a flex layout with wrapping, I don't know that there's a simple way to resolve it.

In a non-GCP hosted scenario, here's what a user would see.
![Screenshot from 2019-05-29 09-01-18](https://user-images.githubusercontent.com/6835846/58559093-57ff6400-81f0-11e9-809c-50f4f414bfbe.png)